lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -1297,4 +1310,168 @@ class StandardAccountTest {
             assertEquals(1, messages[1].order)
         }
     }
+
+    @Nested
+    inner class OutsideExecutionTest {
+        private val randomAddress = Felt.fromHex("0x123")
```

Comment:
I don't think these tests should be wrapped in inner class - note that they belong to `StandardAccount`, so there's no need to wrap it in extra class.